ner_v1/detectors/base_detector.py

```diff
@@ -158,9 +157,84 @@ def detect(self, message=None, structured_value=None, fallback_value=None, **kwa
         else:
             return None
 
+        if value and type(value[0]) is list:
```

Comment:
Like discussed offline this section should move to `detect_entity_bulk()`. Base class will provide a simple implementation of running everything sequentially. Derived classes can provide special implementations for `detect_entity_bulk()`. Also please use plurals in detect_entity_bulk() to make it clear that they operate with lists

Reply:
Done